Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -31,12 +79,21 @@ import Foundation
 
     /// Optional: Item description. Maximum 127 characters.
     public let itemDescription: String? = nil
+    
+    /// Optional: The URL to product information.
+    public let url: URL? = nil
 
     /// Optional: Product or UPC code for the item. Maximum 127 characters.
     public let productCode: String? = nil
+    
+    /// Optional: The URL to product image information.
+    public var imageURL: URL? = nil
 
-    /// Optional: The URL to product information.
-    public let url: URL? = nil
+        /// Optional: UPC code for the item.
+    public var upcCode: String? = nil
+
+        /// Optional: UPC type for the item.
+    public var upcType: BTPayPalLineItemUpcType? = nil
```

Comment:
Optional enums do not exist in Obj-C, take a look at how we handle this with `BTPayPalLocaleCode` and defaulting to a `.none` case in this file and that enums implementation.